INFO : [22/Aug/2023 12:07:04 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:07:04 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 1 validation error for VendorDbDTO
confirm_password
  Input should be a valid string [type=string_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.2/v/string_type]
INFO : [22/Aug/2023 12:07:53 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:07:53 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 1 validation error for VendorDbDTO
confirm_password
  Input should be a valid string [type=string_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.2/v/string_type]
INFO : [22/Aug/2023 12:10:02 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:10:02 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() takes 1 positional argument but 6 were given]
INFO : [22/Aug/2023 12:11:45 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:11:45 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() takes 1 positional argument but 6 were given]
INFO : [22/Aug/2023 12:11:47 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:11:47 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() takes 1 positional argument but 6 were given]
INFO : [22/Aug/2023 12:11:53 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:11:53 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() takes 1 positional argument but 6 were given]
INFO : [22/Aug/2023 12:12:30 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:12:30 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() takes 1 positional argument but 6 were given]
INFO : [22/Aug/2023 12:12:45 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:12:45 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : '"VENDOR"']
INFO : [22/Aug/2023 12:12:51 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:12:51 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : '"VENDOR"']
INFO : [22/Aug/2023 12:13:50 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:13:50 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : '"VENDOR"']
INFO : [22/Aug/2023 12:14:10 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:14:10 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : '"VENDOR"']
INFO : [22/Aug/2023 12:14:20 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:14:20 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() takes 1 positional argument but 6 were given]
INFO : [22/Aug/2023 12:15:35 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:15:35 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() takes 1 positional argument but 6 were given]
INFO : [22/Aug/2023 12:16:02 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:18:11 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:19:14 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:19:37 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:20:24 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:20:24 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'phone']
INFO : [22/Aug/2023 12:21:19 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:21:19 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'phone']
INFO : [22/Aug/2023 12:21:36 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:21:36 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'phone']
INFO : [22/Aug/2023 12:22:06 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:22:06 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'phone']
INFO : [22/Aug/2023 12:22:18 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:22:18 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : uuid3() missing 1 required positional argument: 'name']
INFO : [22/Aug/2023 12:22:42 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:22:42 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'phone']
INFO : [22/Aug/2023 12:23:28 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:25:47 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:27:45 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:27:45 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : Passwords are not same]
INFO : [22/Aug/2023 12:28:15 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:30:16 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:30:26 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:31:06 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:33:09 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:33:09 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : invalid email!]
INFO : [22/Aug/2023 12:33:16 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:33:16 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : invalid email!]
INFO : [22/Aug/2023 12:37:30 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:37:30 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : Passwords are not same]
INFO : [22/Aug/2023 12:41:31 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:45:04 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 12:51:06 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 12:51:06 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 2 validation errors for VendorDbDTO
confirm_password
  String should have at most 8 characters [type=string_too_long, input_value='shaw@2001', input_type=str]
    For further information visit https://errors.pydantic.dev/2.2/v/string_too_long
password
  String should have at most 8 characters [type=string_too_long, input_value='shaw@2001', input_type=str]
    For further information visit https://errors.pydantic.dev/2.2/v/string_too_long]
INFO : [22/Aug/2023 12:52:13 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:02:40 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:21:49 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:21:49 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() got an unexpected keyword argument 'full_name']
INFO : [22/Aug/2023 13:25:34 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:25:34 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() got an unexpected keyword argument 'full_name']
INFO : [22/Aug/2023 13:25:50 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:25:50 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() got an unexpected keyword argument 'full_name']
INFO : [22/Aug/2023 13:26:08 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:26:08 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() got an unexpected keyword argument 'full_name']
INFO : [22/Aug/2023 13:28:07 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:28:07 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() got an unexpected keyword argument 'full_name']
INFO : [22/Aug/2023 13:28:38 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:28:38 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() got an unexpected keyword argument 'full_name']
INFO : [22/Aug/2023 13:30:54 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:30:54 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() got an unexpected keyword argument 'full_name']
INFO : [22/Aug/2023 13:32:34 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:32:34 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'CustomerDbDTO' object does not support item deletion]
INFO : [22/Aug/2023 13:33:02 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:33:02 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'CustomerDbDTO' object does not support item deletion]
INFO : [22/Aug/2023 13:33:19 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:33:19 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'CustomerDbDTO' object does not support item deletion]
INFO : [22/Aug/2023 13:33:55 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:33:55 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'CustomerDbDTO' object does not support item deletion]
INFO : [22/Aug/2023 13:34:22 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:34:22 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'CustomerDbDTO' object does not support item deletion]
INFO : [22/Aug/2023 13:34:46 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:34:46 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'CustomerDbDTO' object does not support item deletion]
INFO : [22/Aug/2023 13:35:26 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:35:26 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : unsupported operand type(s) for |: 'CustomerDbDTO' and 'dict']
INFO : [22/Aug/2023 13:35:49 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:35:49 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : unsupported operand type(s) for |: 'CustomerDbDTO' and 'dict']
INFO : [22/Aug/2023 13:36:23 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:36:23 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'CustomerDbDTO' object has no attribute 'customer']
INFO : [22/Aug/2023 13:37:25 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:37:25 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : unsupported operand type(s) for |: 'CustomerDbDTO' and 'dict']
INFO : [22/Aug/2023 13:39:56 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:39:56 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'dict' object has no attribute '__dict__']
INFO : [22/Aug/2023 13:41:40 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:41:40 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : name 'data' is not defined]
INFO : [22/Aug/2023 13:41:55 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:41:55 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : name 'data' is not defined]
INFO : [22/Aug/2023 13:42:11 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:42:11 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : name 'data' is not defined]
INFO : [22/Aug/2023 13:42:28 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:42:28 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : name 'data' is not defined]
INFO : [22/Aug/2023 13:43:16 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:44:06 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:44:34 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:46:02 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:46:19 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:48:00 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:48:08 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:48:13 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:48:35 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:48:45 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:49:47 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:50:30 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 13:51:15 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 13:51:18 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : {'id': [ErrorDetail(string='customer with this id already exists.', code='unique')], 'email': [ErrorDetail(string='customer with this email already exists.', code='unique')], 'phone': [ErrorDetail(string='customer with this phone already exists.', code='unique')]}]
INFO : [22/Aug/2023 15:39:15 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 15:39:15 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'object' object has no attribute 'data']
INFO : [22/Aug/2023 15:40:10 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 15:40:10 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'object' object has no attribute 'data']
INFO : [22/Aug/2023 15:40:36 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 15:40:36 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'object' object has no attribute 'password']
INFO : [22/Aug/2023 15:41:07 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 15:41:07 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 'VendorDbDTO' object has no attribute 'customer']
INFO : [22/Aug/2023 15:42:13 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 15:42:32 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 15:42:35 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : {'id': [ErrorDetail(string='vendor with this id already exists.', code='unique')], 'email': [ErrorDetail(string='vendor with this email already exists.', code='unique')], 'phone': [ErrorDetail(string='vendor with this phone already exists.', code='unique')]}]
INFO : [22/Aug/2023 15:42:53 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 15:42:56 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : {'email': [ErrorDetail(string='vendor with this email already exists.', code='unique')]}]
INFO : [22/Aug/2023 15:43:03 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 15:53:17 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 15:53:17 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : __init__() got an unexpected keyword argument 'request']
INFO : [22/Aug/2023 16:02:05 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 16:02:05 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : Passwords are not same]
INFO : [22/Aug/2023 16:02:46 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 16:02:46 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : 1 validation error for AdminDTO
permission_level
  Input should be a valid string [type=string_type, input_value=3, input_type=int]
    For further information visit https://errors.pydantic.dev/2.2/v/string_type]
INFO : [22/Aug/2023 16:03:28 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 16:04:58 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 16:05:00 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : {'id': [ErrorDetail(string='admin with this id already exists.', code='unique')], 'phone': [ErrorDetail(string='admin with this phone already exists.', code='unique')]}]
INFO : [22/Aug/2023 16:06:04 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 16:07:20 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 16:07:22 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : {'id': [ErrorDetail(string='admin with this id already exists.', code='unique')], 'phone': [ErrorDetail(string='admin with this phone already exists.', code='unique')]}]
INFO : [22/Aug/2023 16:07:52 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 16:09:28 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 20:01:43 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 20:07:46 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:07:46 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : 2 validation errors for LoginDTO
username
  Field required [type=missing, input_value={}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.2/v/missing
password
  Field required [type=missing, input_value={}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.2/v/missing]
INFO : [22/Aug/2023 20:08:59 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 20:09:24 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 20:16:35 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:16:37 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : Invalid field name(s) given in select_related: 'user_brand_role'. Choices are: user, brand]
INFO : [22/Aug/2023 20:16:58 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:17:00 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : Invalid field name(s) given in select_related: 'user_brand_role'. Choices are: user, brand]
INFO : [22/Aug/2023 20:17:16 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:17:18 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : Invalid field name(s) given in select_related: 'user_brand_role'. Choices are: (none)]
INFO : [22/Aug/2023 20:17:21 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:17:23 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : Invalid field name(s) given in select_related: 'user_brand_role'. Choices are: (none)]
INFO : [22/Aug/2023 20:18:01 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:18:03 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : Invalid field name(s) given in select_related: 'user_brand_role'. Choices are: user, brand]
INFO : [22/Aug/2023 20:18:22 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 20:20:41 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 20:21:23 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 20:22:08 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 20:23:35 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 20:29:43 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:29:43 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : 'VendorLogin' object has no attribute 'is_matched']
INFO : [22/Aug/2023 20:30:04 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 20:35:29 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:35:31 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : 'JwtBuilder' object has no attribute 'payload']
INFO : [22/Aug/2023 20:35:59 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:36:02 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : 'HTTP_ORIGIN']
INFO : [22/Aug/2023 20:37:44 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:37:47 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : 'HTTP_ORIGIN']
INFO : [22/Aug/2023 20:38:06 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:38:09 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : 'JwtBuilder' object has no attribute 'payload']
INFO : [22/Aug/2023 20:38:41 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:38:43 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : 'JwtBuilder' object has no attribute 'payload']
INFO : [22/Aug/2023 20:39:43 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:39:45 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : 'HTTP_ORIGIN']
INFO : [22/Aug/2023 20:40:05 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 20:40:07 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : 'HTTP_ORIGIN']
INFO : [22/Aug/2023 20:40:53 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 20:46:50 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 20:54:13 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 20:56:32 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 21:19:38 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 21:27:40 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 21:28:01 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 21:28:03 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : {'id': [ErrorDetail(string='vendor with this id already exists.', code='unique')], 'email': [ErrorDetail(string='vendor with this email already exists.', code='unique')], 'phone': [ErrorDetail(string='vendor with this phone already exists.', code='unique')]}]
INFO : [22/Aug/2023 21:29:34 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 21:29:37 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : {'id': [ErrorDetail(string='vendor with this id already exists.', code='unique')], 'email': [ErrorDetail(string='vendor with this email already exists.', code='unique')], 'phone': [ErrorDetail(string='vendor with this phone already exists.', code='unique')]}]
INFO : [22/Aug/2023 21:30:25 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 21:30:28 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : {'id': [ErrorDetail(string='vendor with this id already exists.', code='unique')], 'email': [ErrorDetail(string='vendor with this email already exists.', code='unique')], 'phone': [ErrorDetail(string='vendor with this phone already exists.', code='unique')]}]
INFO : [22/Aug/2023 21:32:47 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 21:42:03 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 21:44:00 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 21:44:00 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : BrandRoles matching query does not exist.]
INFO : [22/Aug/2023 21:44:38 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 21:44:39 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : local variable 'brand_role_name' referenced before assignment]
INFO : [22/Aug/2023 21:45:57 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 21:49:10 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 21:49:11 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : value too long for type character varying(350)
]
INFO : [22/Aug/2023 21:49:26 package : micro_auth_service.controller.loginController --> position : Login.post]
ERROR : [22/Aug/2023 21:49:27 package : micro_auth_service.controller.loginController --> position : Login.post --> exception : value too long for type character varying(350)
]
INFO : [22/Aug/2023 21:52:29 package : micro_auth_service.controller.registerController --> position : Register.post]
ERROR : [22/Aug/2023 21:52:31 package : micro_auth_service.controller.registerController --> position : Register.post --> exception : {'refresh_token': [ErrorDetail(string='This field may not be blank.', code='blank')]}]
INFO : [22/Aug/2023 21:53:43 package : micro_auth_service.controller.registerController --> position : Register.post]
INFO : [22/Aug/2023 21:54:16 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 21:54:39 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 21:54:58 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 21:55:21 package : micro_auth_service.controller.loginController --> position : Login.post]
INFO : [22/Aug/2023 21:55:43 package : micro_auth_service.controller.loginController --> position : Login.post]
